packages/control-property-editor/src/slice.ts

```diff
@@ -219,6 +222,9 @@ const slice = createSlice<SliceState, SliceCaseReducers<SliceState>, string>({
                     state.changes.controls[key] = control;
                 }
             })
+            .addMatcher(showMessage.match, (state, action: ReturnType<typeof showMessage>): void => {
+                state.showDialogMessage = action.payload;
```

Comment:
It would be good to rename this variable to dialogMessage. And also in the initialState.

So here when you match showMessage you set the state like this: 
```typescript
state.dialogMessage = action payload;
```

Also in `App.tsx` when retrieving from the store change showDialogMessage to dialogMessage.

Reply:
Done